devicetypes/smartthings/ikea-window-shade.src/ikea-window-shade.groovy

```diff
@@ -0,0 +1,266 @@
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ */
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+	definition(name: "IKEA Window Shade", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.blind", mnmn: "SmartThings", vid: "generic-shade-2") {
+		capability "Actuator"
+		capability "Battery"
+		capability "Configuration"
+		capability "Refresh"
+		capability "Window Shade"
+		capability "Health Check"
+		capability "Switch Level"
+
+		command "pause"
+
+		fingerprint manufacturer: "IKEA of Sweden", model: "KADRILJ roller blind", deviceJoinName: "IKEA KADRILJ Blinds" // raw description 01 0104 0202 00 09 0000 0001 0003 0004 0005 0020 0102 1000 FC7C 02 0019 1000
+		fingerprint manufacturer: "IKEA of Sweden", model: "FYRTUR block-out roller blind", deviceJoinName: "IKEA FYRTUR Blinds" // raw description 01 0104 0202 01 09 0000 0001 0003 0004 0005 0020 0102 1000 FC7C 02 0019 1000
+	}
+
+	tiles(scale: 2) {
+		multiAttributeTile(name:"windowShade", type: "generic", width: 6, height: 4) {
+			tileAttribute("device.windowShade", key: "PRIMARY_CONTROL") {
+				attributeState "open", label: 'Open', action: "close", icon: "http://www.ezex.co.kr/img/st/window_open.png", backgroundColor: "#00A0DC", nextState: "closing"
+				attributeState "closed", label: 'Closed', action: "open", icon: "http://www.ezex.co.kr/img/st/window_close.png", backgroundColor: "#ffffff", nextState: "opening"
+				attributeState "partially open", label: 'Partially open', action: "close", icon: "http://www.ezex.co.kr/img/st/window_open.png", backgroundColor: "#d45614", nextState: "closing"
+				attributeState "opening", label: 'Opening', action: "pause", icon: "http://www.ezex.co.kr/img/st/window_open.png", backgroundColor: "#00A0DC", nextState: "partially open"
+				attributeState "closing", label: 'Closing', action: "pause", icon: "http://www.ezex.co.kr/img/st/window_close.png", backgroundColor: "#ffffff", nextState: "partially open"
+			}
+		}
+		standardTile("contPause", "device.switch", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "pause", label:"", icon:'st.sonos.pause-btn', action:'pause', backgroundColor:"#cccccc"
+		}
+		standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 1) {
+			state "default", label:"", action:"refresh.refresh", icon:"st.secondary.refresh"
+		}
+		valueTile("shadeLevel", "device.level", width: 4, height: 1) {
+			state "level", label: 'Shade is ${currentValue}% up', defaultState: true
+		}
+		controlTile("levelSliderControl", "device.level", "slider", width:2, height: 1, inactiveLabel: false) {
+			state "level", action:"switch level.setLevel"
+		}
+
+		main "windowShade"
+		details(["windowShade", "contPause", "shadeLevel", "levelSliderControl", "refresh"])
```

Comment:
Add a tile for battery like was done in https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/11365/files

Reply:
Would much appreciate this included.

Reply:
I will add that tile in a new zigbee-window-shade-battery DTH.